techniques/system/common/1.0/promises.st

```diff
@@ -188,6 +182,9 @@ bundle common va
        # Need to remove cf_null from the list
        "ncf_inputs"         slist => filter("${ncf_path}/cf_null", "raw_ncf_inputs", "false", "true", 10000);
 
+    # all other inputs are loaded by body file control in rudder-system-directives.cf and rudder-directives.cf
+    "inputs_list" slist => { @{ncf_inputs} };
```

Comment:
This will override the previous definition of inputs_list for "policy_server AND INITIAL_PROMISES" (see line 138)